CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2020-04-08)
+
+- Updated the colorpicker component to implement the affix and overlay services. [#74](https://github.com/blackbaud/skyux-lookup/pull/74)
```

Comment:
Do affix and overlay need to be within tick marks? If not, looks good to me.

Reply:
Wait...this says colorpicker. Is that correct?

Reply:
Oops! Copy-pasted from the other changelog. (The services do not need the tick marks.)

Reply:
Doesn't seem like it. This is the lookup repo, and colorpicker is not one of the components there.